src/FoldersView/FoldersListView.vala

```diff
@@ -56,11 +56,46 @@ public class Mail.FoldersListView : Gtk.Grid {
         var scrolled_window = new Gtk.ScrolledWindow (null, null);
         scrolled_window.add (source_list);
 
+        var load_images_menuitem = new Granite.SwitchModelButton (_("Always Show Remote Images"));
+
+        var account_settings_menuitem = new Gtk.ModelButton () {
+            text = _("Account Settings…")
+        };
+
+        var app_menu_separator = new Gtk.Separator (Gtk.Orientation.HORIZONTAL) {
+            margin_bottom = 3,
+            margin_top = 3
+        };
+
+        var app_menu_box = new Gtk.Box (VERTICAL, 0) {
```

Comment:
shouldn't this be Gtk.Orientation.VERTICAL rather than VERTICAL? to be inline with the rest of the file.
```suggestion
        var app_menu_box = new Gtk.Box (Gtk.Orientation.VERTICAL, 0) {
```

Reply:
I don't think we have anything in the style guide about it. I just found out we could do this recently. IMO, it makes things more legible, but I'm not picky about it